lib/Timber.php

```diff
@@ -261,6 +261,34 @@ public static function get_post( $query = false, $options = [] ) {
 		return $result;
 	}
 
+	/**
+	 * Alias of Timber::get_post(). As such, will honor Class Maps and will NOT
+	 * necessarily always return an Attachment object. Intended for cases where you
+	 * already know you have an attachment ID (or WP_Post of type `attachment`,
+	 * etc.), to clarify your own code.
+	 *
+	 * @api
+	 * @see Timber::get_post()
+	 * @see https://timber.github.io/docs/v2/guides/class-maps/
+	 */
+	public static function get_attachment( $query = false, $options = [] ) {
+		return static::get_post( $query, $options );
```

Comment:
What if we did something like this to handle these cases?

```suggestion
		$post = static::get_post( $query, $options );
		
		return $post && $post instanceof \Timber\Attachment ? $post : false;
```

Wouldn’t that be the *expected* result?

Or wouldn’t that work well either, because somebody could use a class that doesn’t extend `Timber\Attachment`? Or would that be too much on an edge case and we could know that people know that they’re doing?

Reply:
I keep going back-and-forth on this in my head. Either way, we're into edge-case land. I think the advantage is this suggestion is that many of the image/attachment methods are unavailable if the object itself _isn't_ an image/attachment. Thus, probably better to get a `false` than an object that can't be used in the expected way.